Fix vs simulcasts

notSeaSmoke opened the pull request:
fixes simulcast.vpy, and now handles 10 bit video as well.

notSeaSmoke committed:
Update simulcast.vpy

This fixes the bugs in Light's script which broke the script, and also fixes playback of non-8-bit video.

notSeaSmoke committed:
remove comment

LightArrowsEXE commented on mpv/.config/mpv/vs/simulcast.vpy:
```diff
@@ -75,7 +72,11 @@ deband_args: Any = dict(iterations=2, threshold=4, radius=16)
 if video_in_dh <= 810:
     deband_args.update(threshold=3, radius=12)
 
+video_in = depth(video_in, 8)
```
Wouldn't it make more sense to scale up to 16bit?

notSeaSmoke replied:
Done

LightArrowsEXE commented on mpv/.config/mpv/vs/simulcast.vpy:
```diff
@@ -75,7 +72,11 @@ deband_args: Any = dict(iterations=2, threshold=4, radius=16)
+deband = core.std.MaskedMerge(deband, video_in, detail_mask)
+
+final = deband
+final.set_output()
```
Two things here

1) Adding `deband` to a variable here instead of outputting it doesn't really provide any useful functionality. 
2) Add newline kthnx

notSeaSmoke replied:
Fixed

notSeaSmoke committed:
Add requested changes

1. Filter in 16 bit instead of 8
2. Remove unnecessary lines
3. Add newline to end of file

LightArrowsEXE committed:
[mpv] further adjustments to simulcast filtering profile
